tailwind.config.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   purge: [],
   theme: {
     colors: {
+      gold: "#FBDC57", // if we always want gold
```

Comment:
I edited our configuration settings so that we're able to make an element always be gold, regardless of whether a day or night theme is active.

This was useful when setting the border of the modal window notifying users of redirection to Freesound, since we always want that to have a yellow border like it does in the [Figma](https://www.figma.com/file/jeyLWmyn78DjqSmbWcJOEd/Tide?node-id=0%3A1), regardless of whether users have a day or night theme active.

I added this as a change to our configuration settings here in case we need similar functionality in the future.

Reply:
I think we should rename this to match our convention of primary/secondary colors. We might change the color later.

Reply:
Hmm... What would the proper convention be?

`primary: "#FBDC57",` wouldn't work if users' night themes are active, so maybe we could do

`staticPrimaryColor: "#FBDC57",`?

Or, maybe

```
dynamicPrimary: "var(--primary-color)",
primary: "#FBDC57",
```

Reply:
I think we should do `primary: "var(--primary-color)",

Reply:
oh jk, @amilajack already beat me to it

Reply:
After the recent change to the design where we decided that this modal window should always have a border, the special rule in `tailwind.config.js` is no longer necessary, so [I removed it with the latest commit at the time of this writing](https://github.com/amilajack/freesound-player/pull/99/commits/4a47f5fec9bc5527ce5d987f2bcaec0a1248049e#diff-ddc1067f080930bc48865f4660316a14b21bbc9f9ba85975ce8479342b983d2e).